core/bug_api.php

```diff
@@ -544,17 +550,32 @@ function create() {
 					      ' . db_param() . ',' . db_param() . ',' . db_param() . ',' . db_param() . ',
 					      ' . db_param() . ',' . db_param() . ',' . db_param() . ',' . db_param() . ',
 					      ' . db_param() . ',' . db_param() . ',' . db_param() . ',' . db_param() . ')';
-		db_query( $t_query, array( $this->project_id, $this->reporter_id, $this->handler_id, $this->duplicate_id, $this->priority, $this->severity, $this->reproducibility, $t_status, $this->resolution, $this->projection, $this->category_id, $this->date_submitted, $this->last_updated, $this->eta, $t_text_id, $this->os, $this->os_build, $this->platform, $this->version, $this->build, $this->profile_id, $this->summary, $this->view_state, $this->sponsorship_total, $this->sticky, $this->fixed_in_version, $this->target_version, $this->due_date ) );
+		db_query( $t_query, array( $this->project_id, $this->reporter_id, $this->handler_id,
+			$this->duplicate_id, $this->priority, $this->severity, $this->reproducibility,
+			$this->status, $this->resolution, $this->projection, $this->category_id,
+			$this->date_submitted, $this->last_updated, $this->eta, $t_text_id, $this->os,
+			$this->os_build, $this->platform, $this->version, $this->build, $this->profile_id,
+			$this->summary, $this->view_state, $this->sponsorship_total, $this->sticky,
+			$this->fixed_in_version, $this->target_version, $this->due_date ) );
 
 		$this->id = db_insert_id( db_get_table( 'bug' ) );
 
 		# log new bug
 		history_log_event_special( $this->id, NEW_BUG );
 
 		# log changes, if any (compare happens in history_log_event_direct)
-		history_log_event_direct( $this->id, 'status', $t_original_status, $t_status );
+		history_log_event_direct( $this->id, 'status', $t_original_status, $this->status );
 		history_log_event_direct( $this->id, 'handler_id', 0, $this->handler_id );
 
+		# Execute all registered update callbacks
+		while( !empty( $this->update_callbacks ) ) {
```

Comment:
Did you consider having "create_callbacks"?  At least as it appears externally to the calls via add_create_callbacks()?